pkg/api/user/store.go

```diff
@@ -0,0 +1,89 @@
+package user
+
+import (
+	"crypto/sha256"
+	"encoding/base32"
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+
+	ctlvmv1alpha1 "github.com/rancher/harvester/pkg/generated/controllers/harvester.cattle.io/v1alpha1"
+	"github.com/rancher/harvester/pkg/indexeres"
+	pkguser "github.com/rancher/harvester/pkg/user"
+
+	"github.com/rancher/apiserver/pkg/apierror"
+	"github.com/rancher/apiserver/pkg/types"
+	"github.com/rancher/wrangler/pkg/schemas/validation"
+)
+
+const (
+	FieldPassword = "password"
+	FieldUsername = "username"
+)
+
+type userStore struct {
+	types.Store
```

Comment:
can we strip the `password` filed when getting the user by APIs

Reply:
sure